src/css/video-js.scss

```diff
@@ -52,3 +52,7 @@
   border: none;
   visibility: hidden;
 }
+
+.video-js *:focus:not(.focus-visible) {
```

Comment:
does the polyfill require the usage of the `.focus-visible` class? We probably want to have both selectors in that case.

Reply:
Yes, according to the polyfill doc. This class is needed.

Reply:
Ah, cool, didn't realize. Yeah, let's just do both, then eventually we can remove this one. Thanks!

Reply:
If you're going to do this (which makes sense because the focus-visible polyfill can only apply a `.focus-visible` **class**, not a `:focus-visible` **pseudo-class**, I think you need to include the syntax I had in my example:

````css
.js-focus-visible .video-js *:focus:not(.focus-visible) {
  outline: none;
}
````

Without the `.js-focus-visible` it's going to remove the focus outline if the developer **doesn't** include the polyfill, which is exactly what we want to avoid.

Reply:
^^^ That comment should have been a review